lib/resources/dstu2/immunization.yaml

```diff
@@ -0,0 +1,77 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  example: |
+            {"coding": [
```

Comment:
A few things

1. We have been including the field name as the initial value in the example hash. I know these won't show up until the field table is added, but we should be consistent so that when the rest of the values are added to this file they are added correctly. 
2. I'm guessing 34840-9 is left over from a copy-paste from DocumentReference, unless you are trying to say that they have a Rheumatology Note vaccine :)

```
{"vaccineCode": {
   "coding": [
     {"system": "http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
      "code": "43"}],
   "text": "Engerix-B (obsolete)"}}
```